std/build.sbt

```diff
@@ -1 +1,5 @@
 name := "std"
+
+libraryDependencies in ThisBuild ++=
```

Comment:
Maybe not?

At the time it seemed like a good idea but it's becoming clear that I don't know what I'm doing.